### PR TITLE
MVP for Class 04

### PR DESCRIPTION
**Pull request by elaewin**

### CODE REVIEW

**Comment by elaewin** [comment]
#### How long the assignment took.

All together, I spent about six hours on this last night.

#### Reflection on the assignment:

This took way, way too long. I really had to step back and work on my breakpoints, since I'd been concentrating exclusively on the smaller version.

I'd been trying to get two big images on the 'home' screen, but decided that I would put that off for now, since the images weren't really intuitive to anyone who wasn't me. My hamburger menu also wasn't working properly on anything but a touch screen interface. 

Fixing the hamburger menu took a reassuringly short time (once I had the right selectors in jQuery), and swapping the hamburger for an 'X' was also very straightforward. However, trying to get the menu to work on a larger screen took a **LOT** of trial and error, and was incredibly frustrating. After some advice from Rick this morning, I actually changed my breakpoint from 640 to 880 so that the big remaining display issue was fixed.

I got the handlebars template working on my site, but it broke my filter, so I need to work on getting that fixed. I also spent some time looking at fonts, but decided that I was too tired to make reasonable decisions (not Comic-Sans-looks-good-tired, but still, too tired.)

Notes to self: MAKE SURE that when I make changes in the console/elements panel I have **all** of them copied over before refreshing the page. FFS.

File: index.html [hunk 1]
```diff
@@ -5,36 +5,57 @@
...
+          <li class="tab" data-content="posts"><a href="#">Blog</a></li>
+          <li class="tab" data-content="bookbinding">My Books</li>
           <li class="tab" data-content="about"><a href="#">About</a></li>
         </ul>
       </nav>
     </header>
     <div class="clearfix"></div>
     <main>
-      <section id="home" class="tab-content">
+      <!-- <section id="home" class="tab-content">
```
Comment (nhuntwalker) [inline index.html:47]: Is this commented block going to be used for something? If not, delete.

File: js/postView.js [hunk 2]
```diff
@@ -6,38 +6,38 @@ postView.handleMainNav = function() {
   $('.main-nav').on('click', '.tab', function() {
     var $selectedTab = $(this).attr('data-content');
     $('section.tab-content').hide();
-    $('section.tab-content').each(function() {
-      if($(this).attr('id') === $selectedTab) {
-        $(this).fadeIn();
-      }
-    });
+    $('section.tab-content[id="' + $selectedTab + '"]').fadeIn();
+    // $('section.tab-content').each(function() {
```
Comment (nhuntwalker) [inline js/postView.js:10]: As with the HTML, remove commented code (unless you're saving it for something)

File: js/postView.js [hunk 3]
```diff
@@ -6,38 +6,38 @@ postView.handleMainNav = function() {
...
     }
   });
 };
 
 postView.handleCategoryFilter = function() {
   $('#category-filter').on('change', function() {
-    if($(this).val()) {
-      var $cat = $(this).val();
+    var $cat = $(this).val();
+    console.log($cat);
```
Comment (nhuntwalker) [inline js/postView.js:35]: Console messages shouldn't be in production code.

**Pull request merged**